.github/workflows/reusable_trivy.yml

```diff
@@ -19,13 +19,20 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Manual Trivy Setup
+        uses: aquasecurity/setup-trivy@9ea583eb67910444b1f64abf338bd2e105a0a93d
```

Comment:
pls add a comment which version of the action is it (e.g. similarly to the one below: `# v0.30.0`)

Reply:
It appears that we are now using the correct version by default, and I no longer see the error in the logs; therefore, I decided to remove the manual setup.